tests/unit/test_scheduler.py

```diff
@@ -0,0 +1,22 @@
+"""Module to test the scheduler.py"""
+from unittest import TestCase
+from unittest.mock import MagicMock
+from apscheduler.jobstores.base import JobLookupError
+from napps.amlight.sdntrace_cp.scheduler import Scheduler
+
+
+class TestScheduler(TestCase):
+    """Test class Scheduler"""
+
+    def test_remove_job_fail(self):
+        """Test remove_job log with non existent id"""
+        scheduler = Scheduler()
+        with self.assertRaises(JobLookupError):
```

Comment:
@Alopalao let's handle `JobLookupError` on `remove_job`, feel free to return `None` if it can't remove the job. That way this abstraction simplifies a bit when for callers calling `remove_job` without having to handle this, and for cases where they care if it indeed if was removed they check if it's not None.

Reply:
`remove_job()` from `BackgroundScheduler` returns `None` also when it successfully runs. Could it return a boolean instead? `False` when it fails, `True` otherwise.

Reply:
@Alopalao, good point. Let's keep it always returning `None` then, so it behaves idempotently, that way it doesn't diverge much from he underlying API. In the future, we can try to raise other errors if ever needed.

Reply:
Got it.